### PR TITLE
linux service instructions

### PR DESCRIPTION
**Pull request by trevarj**
- Added install command to Makefile
- Added systemd service and instructions

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,3 +26,7 @@ windows: $(OBJ) $(WIN_OBJ)
 
 clean:
 	rm -f $(TARGET) $(TARGET).exe $(OBJ) $(WIN_OBJ)
+
+install: $(TARGET)
+	mkdir -p $(INSTALL_DIR)
+	install $(TARGET) $(INSTALL_DIR)
```
Comment (dartvader316) [inline Makefile:32]: `install -m 755` would be good to use here

File: dist/linux/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+make
+sudo make install
+```
+
+## Systemd Service (optional)
+
+Copy and enable the service:
+
+```sh
+cp byedpi.service /etc/systemd/system/byedpi.service
```
Comment (dartvader316) [inline dist/linux/README.md:15]: Systemd supports user services in `~/.config/systemd/user/`. 
byedpi does not require root privileges to run. So i think user service will be much safer.

Comment (trevarj) [reply dist/linux/README.md:15]: good point. thanks!

**Review by dartvader316 — APPROVED** [review]
LGTM

Ping @hufrea for merge.

File: dist/linux/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+sudo make install
+```
+
+## Systemd Service (optional)
+
+Copy and enable the service:
+
+```sh
+cp byedpi.service ~/.config/systemd/user/
+sudo systemctl daemon-reload
```
Comment (hufrea) [inline dist/linux/README.md:16]: `systemctl --user daemon-reload`

Comment (trevarj) [reply dist/linux/README.md]: thanks. fixed.

**Pull request merged**